spotify-artist-viewer/ArtistTableViewCell.m

```diff
@@ -0,0 +1,36 @@
+//
+//  ArtistTableViewCell.m
+//  spotify-artist-viewer
+//
+//  Created by Randall Spence on 6/10/15.
+//  Copyright (c) 2015 Intrepid. All rights reserved.
+//
+
+#import "ArtistTableViewCell.h"
+#import "SAArtist.h"
+
+@implementation ArtistTableViewCell
+
+- (void)awakeFromNib {
+    // Initialization code
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
+    [super setSelected:selected animated:animated];
+
+    // Configure the view for the selected state
+}
+
+-(void)configureCell:(UITableViewCell*)cell ForForm:(SAArtist *)cellArtist{
+    ArtistTableViewCell* newCell = [[ArtistTableViewCell alloc] init];
+    newCell.textLabel.text = cellArtist.name;
+    cell = newCell;
+    
```

Comment:
Instead of creating a new call here just configure self. For this method you really only need self.textLabel.text = cellArtist.name

For clarity probably best to just call the Artist param artist, not cellArtist